contracts/src/ethereum/TokenBridge.sol

```diff
@@ -133,16 +137,17 @@ contract TokenBridge is
 
     function sendMessageStaticAToken(address l1Token, uint256 accRewards) 
```

Comment:
This can be wrapped with the modifier instead

Reply:
or we can verify that the provided accRewards is valid (by getting the current rewards index on `staticAToken` but that would entail some extra gas)

Reply:
@hsouf we could alternatively restrict the call to l1 staticAToken